arches/management/commands/updateproject.py

```diff
@@ -262,7 +286,10 @@ def update_to_v7_6(self):
                 file.close()
 
                 updated_file_data = (
-                    file_data.replace("{{ project_name }}", settings.APP_NAME)
+                    file_data.replace(
+                        "{{ project_name_title_case }}", settings.APP_NAME.title()
```

Comment:
why is this / verbose_name in `apps.py` needed?

Reply:
I'm doing this string replacement in two places:
- one, the class name for the AppConfig class, e.g. DiscoConfig. If the project name is "disco", we don't want a class called "discoConfig", we want "DiscoConfig".
- two, for the `verbose_name`: it's shown in all the [basic doc'd examples](https://docs.djangoproject.com/en/5.0/ref/applications/). We could decide to remove it, since it defaults to label.title() anyway, but I liked the idea of leaving it in to "leave it in the user's face" so they might consider using a better one if it looks wrong, e.g. "Arches for Science" instead of "Arches_For_Science" (default)